<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(34, 32, 34)</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QToolBox" name="toolBox">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(49, 47, 49);
color:white
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="midLineWidth">
       <number>42</number>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>432</width>
         <height>530</height>
        </rect>
       </property>
       <attribute name="label">
        <string>                                              Настройка COM-порта</string>
       </attribute>
       <widget class="QLabel" name="setComPort">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>92</width>
          <height>13</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Выбор COM-порта</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxCom">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>191</width>
          <height>41</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
       <widget class="QLabel" name="setBaudRate">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>81</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Выбор скорости</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxBaudRate">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>191</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="setStopBits">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>260</y>
          <width>47</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Стоп-бит</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxStopBits">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>280</y>
          <width>191</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxParity">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>191</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="setParityBits">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>190</y>
          <width>68</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Бит четности</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_4">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>430</y>
          <width>191</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(89, 89, 89)</string>
        </property>
        <property name="text">
         <string>Обновить</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxFlowControl">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>360</y>
          <width>191</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="setFlowControl">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>340</y>
          <width>88</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Контроль потока</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxDataBits">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>430</y>
          <width>191</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>140</y>
          <width>191</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(241, 0, 2)</string>
        </property>
        <property name="text">
         <string>Подключить порт</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_6">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>210</y>
          <width>191</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(89, 89, 89)</string>
        </property>
        <property name="text">
         <string>Отключить порт</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_7">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>60</y>
          <width>191</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(89, 89, 89)</string>
        </property>
        <property name="text">
         <string>Сохранить настройки</string>
        </property>
       </widget>
       <widget class="QLabel" name="setDataBits">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>410</y>
          <width>159</width>
          <height>13</height>
         </rect>
        </property>
        <property name="text">
         <string>Количество передаваемых бит</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>432</width>
         <height>530</height>
        </rect>
       </property>
       <attribute name="label">
        <string>                                              Ключи и Шифрование</string>
       </attribute>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>181</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>MS Shell Dlg 2</family>
          <pointsize>10</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(89, 89, 89)</string>
        </property>
        <property name="text">
         <string>СГЕНЕРИРОВАТЬ КЛЮЧ</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>181</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(89, 89, 89)</string>
        </property>
        <property name="text">
         <string>ЗАГРУЗИТЬ КЛЮЧ</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>181</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(89, 89, 89)</string>
        </property>
        <property name="text">
         <string>ПРОЧИТАТЬ КЛЮЧ</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>270</y>
          <width>411</width>
          <height>251</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>190</y>
          <width>411</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(0, 0, 0)</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>432</width>
         <height>530</height>
        </rect>
       </property>
       <attribute name="label">
        <string>                                               Параметры гарнитуры</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
